public/media-hunt/index.js

```diff
@@ -1 +1,27 @@
-const app = require("./mediahunt.js");
```

Comment:
Please revert changes on this file, or rebase from master to fix conflict.

Reply:
resolved